### PR TITLE
ON-5946 # Add logged in user information route

### PR DESCRIPTION
**Pull request by kizaonline**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+install:
+  - npm install --global npm@6.x.x
+  - npm install -g .
```
Comment (mymattcarroll) [inline .travis.yml:17]: Prob dont need this

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
```
Comment (mymattcarroll) [inline CHANGELOG.md:4]: This project can not adhere to semantic versioning as there is only one published product.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+   ```
+   git push && git push --tags
+   ```
+
+1. Publish changes to npm
+
+   ```
+   npm publish
+   ```
```
Comment (mymattcarroll) [inline CONTRIBUTING.md:67]: We are not publishing this to npm, right? Are you updating this in the next task?

File: src/routes/logged-in-user-information.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+      }
+      return user
+    } else {
+      return {}
+    }
+  } catch (error) {
+    console.error(error)
+    res.setStatusCode(500)
+    res.setPayload({ message: error })
+  }
```
Comment (mymattcarroll) [inline src/routes/logged-in-user-information.js:96]: Catch is redundant, this is exactly what the API CLI wrapper does :)

File: CHANGELOG.md (hunk 2 above)
Comment (kizaonline) [reply CHANGELOG.md:4]: Why cant it?

File: src/routes/logged-in-user-information.js (hunk 4 above)
Comment (kizaonline) [reply src/routes/logged-in-user-information.js:96]: I couldnt find this anywhere in the cli docs

Comment (mymattcarroll) [reply src/routes/logged-in-user-information.js:87]: Good call, ill add to this ticket: https://blinkmobile.atlassian.net/browse/ON-5705

File: src/routes/logged-in-user-information.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+function decodeToken(token /* :string | void */) {
+  if (token) {
+    const { payload } = jsonwebtoken.decode(token, { complete: true })
+    return payload
+  }
+}
+
+module.exports = getBearerToken
```
Comment (mymattcarroll) [inline src/routes/logged-in-user-information.js:29]: prob want to remove this.

File: CHANGELOG.md (hunk 2 above)
Comment (mymattcarroll) [reply CHANGELOG.md:4]: Users cannot pick a version, hence semantic versioning is redundant. They have have to use the latest version.

Comment (kizaonline) [reply CHANGELOG.md:4]: Same argument could be made for the rest of the Productivity Suite...

**Pull request merged**